HW 6/06_functions_2.py

```diff
@@ -0,0 +1,8 @@
+# Task 2 - Create a function which sums all the arguments and returns the result
+def sum_all(*args):
+    return sum(args)
+print(sum_all(9, 6, 66, 776))
+
+# sum_number = float(input("Enter the number to sum: "))
+# result = sum_all(sum_number)
+# print("Total: ", result)
```

Comment:
I think you wanted to enter several numbers separated by commas like? "1, 3, 7, 1"
```suggestion
user_data = input("Enter the number to sum: ")
numbers = [int(item) for item in user_data.split(", ")]
result = sum_all(sum_number)
print("Total: ", result)
```